executor/builder.go

```diff
@@ -167,7 +167,7 @@ func (b *executorBuilder) buildTableScan(v *plan.TableScan) Executor {
 		seekHandle:  math.MinInt64,
 	}
 	x := b.buildFilter(e, v.FilterConditions)
-	if v.Desc && !supportDesc {
```

Comment:
Because our old plain executor `TableScan` and `IndexScan` doesn't support descending order.